src/sub-blocks/BasicCard/BasicCard.tsx

```diff
@@ -41,13 +44,26 @@ const BasicCard = (props: BasicCardProps) => {
                         text={text}
                         textId={descriptionId}
                         additionalInfo={additionalInfo}
-                        links={links}
-                        buttons={buttons}
+                        links={controlPosition === 'content' ? links : undefined}
+                        buttons={controlPosition === 'content' ? buttons : undefined}
                         size="s"
                         colSizes={{all: 12, md: 12}}
                     />
                 </IconWrapper>
             </CardBase.Content>
+            <CardBase.Footer className={b('footer')}>
```

Comment:
confused a littler that we should copy-paster this 3 times, could we create component for this?